src/nvim/main.c

```diff
@@ -834,20 +808,25 @@ void getout(int exitval)
   mch_exit(exitval);
 }
 
-/*
- * Get a (optional) count for a Vim argument.
- */
-static int
-get_number_arg (
-    char_u *p,             /* pointer to argument */
-    int *idx,           /* index in argument, is incremented */
-    int def                    /* default value */
-)
+/// Gets the integer value of a numeric command line argument if given,
+/// such as '-o10'.
+///
+/// @param p pointer to argument
+/// @param idx pointer to index in argument
```

Comment:
That looks a bit awkward, since the `o` in `out]` is lined up with the `p` above it, but I'll do that for now.